docs/interacting-with-openshift.md

```diff
@@ -99,48 +99,58 @@ $ minishift logs
 <a name="updating-openshift-configuration"></a>
 ## Updating OpenShift configuration
 
-Once OpenShift is running, you can view and change the master and
-node configuration of your OpenShift cluster.
+While OpenShift is running, you can view and change the master or the node configuration of your OpenShift cluster.
 
-You can view the current OpenShift master configuration (_master-config.yaml_) via:
+To view the current OpenShift master configuration (_master-config.yaml_), run the following command:
 
 ```shell
 $ minishift openshift config view
 ```
 
-For displaying the node configuration, you can specify the `target` flag.
-For more details about the `view` command refer to its [synopsis](./minishift_openshift_config_view.md).
+To show the node configuration instead of the master configuration, specify the `target` flag.
 
-Let's look at [Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS) as an example for patching the OpenShift master configuration.
-Per default, OpenShift will only allow cross origin resource requests from the IP of the
-cluster as well as localhost. This is specified via the `corsAllowedOrigins` property in the
-[master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_). To change this value and allow
-cross origin requests from all domains, one can execute:
+For details about the `view` command, see the [synopsis](./minishift_openshift_config_view.md) command reference.
+
+### Use-case examples
+
+#### Configuring cross-origin resource sharing
+
+In this use-case example, you update the OpenShift master configuration to change the resource IP
```

Comment:
In this use-case example, you update the OpenShift master configuration to change the IP addresses from which cross-origin requests are allowed (see [cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS)).

"change the resource IP" is imo not correct.